tech-specs/src/explore/design/dkg.md

```diff
@@ -1 +1,178 @@
-# Distributed key generation gossip
+# Distributed key generation
+
+Here we give an overview of how the DKG protocol works with the ledger and 
+the roles of each node in the network per this protocol. This portion of the 
+book focuses solely on the production of a new public encryption key via the
+DKG scheme. It does not cover encryption, decryption, or validation of transactions.
+
+Some parts of the DKG protocol may be done through the gossip layer or through
+off-chain communication. However, the entire DKG cannot exist solely in the 
+gossip layer as consensus on the final shared secrets must be achieved. This 
+will require posting information on chain.
+
+## Starting the Protocol
+
+The protocol to generate a new key will be scheduled to begin at specific 
+block heights, in pratices this will be at the beginning of every eposh.
+This way all the validators know when to begin the announcement phase. In 
+addition to starting at the pre-determined blocks heights, if the protocol 
+fails, all validators know to immediately start a new round of the protocol.
```

Comment:
assuming that the validators can detect that the protocol has failed, we should clarify this

Reply:
At this point, I'm not sure the protocol can fail. We just keep waiting until it finishes. I suppose it hits a natural "timeout" if somehow it lasts a whole epoch, but that shouldn't realistically ever happen.

Reply:
what do you mean by "if the protocol fails" then?

Reply:
It actually doesnt say anything about failing anymore. I should have resolved this conversation when I changed that.